pages/linux/tmt.md

```diff
@@ -0,0 +1,37 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> See also: `tmt-run`, `tmt-try`.
```

Comment:
```suggestion
> Some subcommands such as `run`, `try`, etc. have their own usage documentation.
```
For subcommand mention we use this syntax:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md